src/ui/components/atoms/Ghost-Button/ghost-button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import type { GhostButtonProps } from './types/types';
+
+// Cambiado a declaración de función
+function GhostButton({
+  ghostbutton,
+  variant,
+  size = 'medium',
+  disabled = false,
+  onClick,
+}: GhostButtonProps) {
+  const buttonClass = [
+    styles['ghost-button'],
+    styles[`ghost-button--${variant}`],
+    styles[`ghost-button--${size}`],
+    disabled ? styles['ghost-button--disabled'] : '',
+  ].join(' ');
```

Comment:
please replace this approach for this https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx#L7-L10

Reply:
fixed engineer